docs/content/foundations/typography/list-reset.mdx

```diff
@@ -10,9 +10,11 @@ utility_modules:
     utility_module_name: 'add-list-reset'
     utility_class_intro: |
       The list reset utility classes have not been modified from the U.S. Web Design System (USWDS). Refer to the [USWDS List reset utility](https://designsystem.digital.gov/utilities/list-reset/#list-reset-2).
-    utility_class_display_component: 'TestUtilityClassTable'
+    utility_class_display_component: 'ListResetDemonstration'
     utility_class_display_params:
-      classes: []
+      classes:
+        - class_name: 'default'
```

Comment:
`.default` is not a class.

Reply:
`.default` on USWDS seems to just be a way for them to show the difference between the list reset and a standard list. I should remove this ?

Reply:
![Screenshot 2024-07-29 at 1 48 39 PM](https://github.com/user-attachments/assets/68dbeecd-9c4c-48a3-b19a-4ddef3b89d91)

You are telling a user that a class `.default` exists. It does not. Our documentation should probably not be lying to users trying to use it.

Reply:
![Screenshot 2024-07-29 at 1 50 54 PM](https://github.com/user-attachments/assets/0bc9d1fc-81d3-4921-acaf-7fad5aa68e0e)

USWDS does not have a class here they are just showing it as an example. but notice how there is no classes